src/ameba/rule/lint/logical_without_paren.cr

```diff
@@ -0,0 +1,55 @@
+module Ameba::Rule::Lint
+  # A rule that disallows logical operators in method args without parenthesis.
+  #
+  # For example, this is considered invalid:
+  #
+  # ```
+  # if a.includes? "b" && c.includes? "c"
```

Comment:
I'd prefer the example from the OP, as it's more clear why it might be bug-prone:

```suggestion
  # if foo.includes? "bar" || foo.includes? "batz"
```

ditto other examples